pxr/base/tf/hash.h

```diff
@@ -94,6 +97,33 @@ TfHashAppend(HashState &h, std::vector<T> const &vec)
     h.AppendContiguous(vec.data(), vec.size());
 }
 
+// Support std::set
+template <class HashState, class T>
+inline void
+TfHashAppend(HashState& h, std::set<T> const &elements)
+{
+    h.AppendRange(std::begin(elements), std::end(elements));
+}
+
+// Support std::map
+// NOTE: Supporting std::unordered_map is complicated because the traveral
```

Comment:
Typo: "traveral" -> "traversal"

It might be worth adding a similar comment above std::set regarding std::unordered_set.